### DISCUSSION TITLE
"HC is offline" overnight while everything is ok

### DISCUSSION THREAD

**Discussion by Dayk0** [mozilla/hubs#3792] [Hubs Cloud Questions] [answered]
Hello, I wake up this morning to go to my business server and I have a big problem. I have the black page with "Hubs Cloud is currently offline. Check back shortly." for no reason. I looked at my battery and everything is green. I have updated several times changing the server type, their numbers and even putting it online and offline.
During the night, I noticed that my server's CPU suddenly increased for no reason. It remained at 36% for 3 hours.
Anyway, it's really strange because my other personal server is working very well. http://snap.ashampoo.com/RpE7i4QF3vpzToyRzs6RtY0ky4pwY0AlOFJxc168QnXf3QxVeUixOiCifR08RakB

What do I have to do ? Delete my current stack then recreate a new stack restoring my server data from an automatic backup last night? I hope the restore will work well because I have a lot of data.
I have an event next week, hope that doesn't happen ...

Thank you for your reply.

**EDIT : It's been 2 hours that I tried to solve the problem and while I was writing this topic, it is corrected for no reason. I would like to know why there was this problem, are there any server or domain updates overnight causing this blockage? I saw that some had had this problem ... so maybe it is not to be taken lightly.**

**Comment by robinkwilson** [comment]
No software updates on Mozilla's end. We're going to keep an eye on this. thanks for the screenshot!

**Can you give me some more information about your hub?**

1. Is this an Enterprise multi-server Hubs Cloud setup?
2. How many app / streaming servers do you have?
3. Did anyone connect during the CPU spike?
4. Is there anything else relevant to add?

**Reply by Dayk0** [reply]
No problem, therefore, the problem occurred overnight French time, I will answer your questions and then explain precisely what happened.

1. Is this a multi-server enterprise Cloud Hubs configuration? **Yes.**
2. How many application / streaming servers do you have? **There were 2 application servers and 2 streaming in t3.micro (just for development, no uses)**
3. Did someone log in during the CPU spike? **Yes, according to Analytics, one person.**
4. Is there anything else to add? **Yes, I stumbled across a forum where people were talking about server bugs that night where it was impossible to connect (it was website servers).**

**To recap the situation, here's what exactly happened:

-During the night of January 27 to 28 (French time), there was a server crash, see image of CPU usage (I don't have better quality because I deleted my tracking on cost manager in around 10h00 and I no longer have the stats from before ...):
http://snap.ashampoo.com/2grLNC7LeZe5NBfwmMBSPh4Jxvf7HFpDLbpzengiiKfVMKMzxOMiwJRoaWmH64lj


- Note that from 8h precise, on the CPU, there was a big drop in the server then it rose to 32% to stagnate until around 10:30 am. It was then that I realized I had the HC server screen idle while all the lights were green on my batteries. However, by going to see on route 53, it indicated to me that the waiters were not healthy. http://snap.ashampoo.com/fs2YBh7Aevt5CocRIlvDv2Ow6MpjWX74lYVMbuNfkBCnqE0YPv5gQm6OYnXqFwnQ

-In addition, by going to Google Analytics, I saw that a person (probably a robot) quickly appeared on the server at 8 a.m., the moment when the servers started to stagnate at 32%.
http://snap.ashampoo.com/AxurBC2NaNbYubDSdbFXwAHnq9o7LtQmlF8niz3ii6c68FqcGVZ7wcLHBZgnoWZz

-Finally, I had changed the number of servers, put it online or offline, but nothing changed. that is to unlock automatically without any reason.**

**Comment by robinkwilson** [comment, marked as answer]
Ah, can you link the forum? Is this forum for AWS? 

**Can you try running t3.small instances at least instead?** You can scale your instance up before and during your event to something larger, then scale it back down after.

**We don't recommend people use t3.micros because the memory for those instances are extremely small and results in unexpected side effects.** We're keeping an eye on this and please post again if you see this happen with larger instance sizes: https://github.com/mozilla/hubs-cloud/issues/180

**Reply by Dayk0** [reply]
I have never had this problem again since. I think indeed, it was because I was in t3.micros then in t3.small. Now I stay in t3.medium for more security.
